modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1054,6 +1061,16 @@ private void cleanUpTablesResources(Map<UUID, TableImpl> tables) {
 
                 CompletableFuture<Void> removeFromGcFuture = mvGc.removeStorage(replicationGroupId);
 
+                stopping.add(() -> {
+                    try {
+                        closeTracker(internalTable.getPartitionSafeTimeTracker(partitionId));
+
+                        closeTracker(internalTable.getPartitionStorageIndexTracker(partitionId));
```

Comment:
Let's extract these lines to a method, they are duplicated a few times

Reply:
Why not.